packages/form-js-playground/src/Playground.js

```diff
@@ -29,7 +29,7 @@ import { PlaygroundRoot } from './components/PlaygroundRoot';
 /**
  * @param {FormPlaygroundOptions} options
  */
-export default function Playground(options) {
+export function Playground(options) {
```

Comment:
I'm a big fan of doing exports only in the last line of the file like `export {Playground}`, that's what we use on Tasklist

This way it's faster everything that a file exports, it would be cool to have the same standards across both projects, but I'm also ok if you want to leave it as is

Reply:
We can do this in a second step if you like, if we can find a linting rule for it.

Reply:
Didn't take long to find, https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/exports-last.md.

I can add another PR once we've got all 3 blocks of changes into a single PR